### PR TITLE
Combine error table

### PR DESCRIPTION
**Pull request by dimasciput**
Fix https://github.com/hotosm/field-campaigner/issues/309

### CODE REVIEW

File: flask_project/campaign_manager/static/js/detail/insight_functions.js [hunk 1]
```diff
@@ -687,7 +700,7 @@ function renderInsightFunctionsTypes(username) {
         index++;
     }
 
-    getOSMCHAErrors();
+    // getOSMCHAErrors();
```
Comment (cchristelis) [inline flask_project/campaign_manager/static/js/detail/insight_functions.js]: rather remove than comment out.

**Comment by cchristelis** [comment]
Hi @dimasciput, it would be great if you added a screenie here as well as comment on how you ordered the errors from the two sources...

**Comment by dimasciput** [comment]
@cchristelis , here's what it looks like 

screen shot 2017-09-26 at 9 07 05 am

**Comment by cchristelis** [comment]
LGTM

**Comment by PaulUithol** [comment]
Are we generating those messages? Some could be clearer. E.g. `Error:
capacity not found` to `Error: key "capacity" not present`, and don't get
what `ceramic to pottery is the accepted tag` would mean?

~ sent from mobile

**Pull request merged**